### ISSUE TITLE
plugin not working with Grails 2.1.1

### ISSUE THREAD

**Issue by gernotstarke** [sarbogast/grails-bootstrap-file-upload#9]
does not even compile, due to errors in svn plugin...
(which is required by twitter-bootstrap for whatever strange reasons)

or please let us know that you cannot fix it currently... 

thx,
Gernot

**Comment by dinizssa** [comment]
Yeah, it was not reviewed for 2.1.1. With regards "svn", I just cleanup the classes with errors, kept the file, but empty. Fixed the compilation issue. That's really bad, plugin with no support and with a high level of dependency (unnecessary). Also, it has different behaviors on Chrome, IE and Firefox (none of them it works properly, probably due to out-of-dated JavaScript calls).

We have 3 options: 1. wait for the author do something (don't reply even e-mails). 2. fix ourselves. 3 give a try using another plugin.

**Comment by EpicVoyage** [comment]
Open plugins\bootstrap-file-upload-2.1.1\dependencies.groovy. Change

```
compile ":twitter-bootstrap:2.0.2.25"
```

to

```
compile ":twitter-bootstrap:2.0.2.25" {excludes 'svn'}
```

Now if there is a way to make this happen automatically...
